src/modules/apps/install.js

```diff
@@ -8,33 +8,44 @@ import {
   wildVersionPattern,
 } from '../../manifest'
 
+const ARGS_START_INDEX = 2
+
+function installApps (apps) {
+  const app = apps.shift() || `${manifest.vendor}.${manifest.name}@${manifest.version}`
+  log.debug('Starting to install app', app)
+  const appRegex = new RegExp(`^${vendorPattern}.${namePattern}@${wildVersionPattern}$`)
+  if (!appRegex.test(app)) {
+    log.error('Invalid app format, please use <vendor>.<name>@<version>')
+    return Promise.resolve()
+  }
+
+  return installApp(app)
+  .then(() => log.info(`Installed app ${app} successfully`))
+  .then(() =>
+    apps.length > 0
+      ? installApps(apps)
+      : Promise.resolve()
+  )
+  .catch(err => {
+    if (err.statusCode === 409) {
+      return log.error(`App ${app} already installed`)
+    }
+    return Promise.reject(err)
+  })
+}
+
 export default {
   optionalArgs: 'app',
   description: 'Install an app on the current directory or a specified one',
-  handler: (optionalApp) => {
+  handler: (optionalApp, options) => {
     const workspace = getWorkspace()
     if (workspace === 'master') {
       log.error(workspaceMasterMessage)
       return Promise.resolve()
     }
 
-    const app = typeof optionalApp === 'string'
-      ? optionalApp
-      : `${manifest.vendor}.${manifest.name}@${manifest.version}`
-    log.debug('Starting to install app', app)
-    const appRegex = new RegExp(`^${vendorPattern}.${namePattern}@${wildVersionPattern}$`)
-    if (!appRegex.test(app)) {
-      log.error('Invalid app format, please use <vendor>.<name>@<version>')
-      return Promise.resolve()
-    }
-
-    return installApp(app)
-    .then(() => log.info(`Installed app ${app} successfully`))
-    .catch(err => {
-      if (err.statusCode === 409) {
-        return log.error(`App ${app} already installed`)
-      }
-      return Promise.reject(err)
-    })
+    const apps = options._ ? [optionalApp, ...options._.slice(ARGS_START_INDEX)] : [optionalApp]
```

Comment:
Assuming `options._` exists. Not necessarily true! I'd rather receive `...apps, options`, no? Does that even work? 😁